### PR TITLE
Philipp lig 1202 clean up docs integration

### PR DESCRIPTION
**Pull request by philippmwirth**
# Philipp lig 1202 clean up docs integration

Closes [LIG-1202](https://linear.app/lightly/issue/LIG-1202/clean-up-docs-integration).

Drops most of the integration info and refactors the dagster tutorial

### CODE REVIEW

**Comment by MalteEbner** [comment]
This PR is quite hard for me to understand. Perhaps you could update the issue description to reflect what this PR really does?

Do I assume the following correctly about what this PR does?
1. Refactors the dagster tutorial to use the workflow of triggering the Lightly Docker over the API
2. Deletes the s3fuse tutorial with replacement, as it is deprecated
3. Deletes the docker_with_datasource and docker_trigger from API. This is part of a move to first_steps. They have already been created in first_steps by this PR: https://github.com/lightly-ai/lightly/pull/821

**Comment by philippmwirth** [comment]
> 

The whole idea of this `rework-docs` branch is to only portrait a single workflow and make it the default: Do everything with Python scripts. For the `integration` section this means that we can drop most of the chapters because they are covered in the "Setup" and "Getting Started" sections. The only leftover is the `dagster` tutorial which I adapted such that it works with the new workflow.

File: docs/source/docker/integration/dagster_aws.rst [hunk 1]
```diff
@@ -2,33 +2,25 @@
...
 
 Data Pre-processing Pipeline on AWS with Dagster
-===================================================
+================================================
 
 
 Introduction
 --------------
-Data collection and pre-processing pipelines have become more and more automated in the recent years. The Lightly Docker can take on a crucial role
+Data collection and pre-processing pipelines have become more and more automated in the recent years. The Lightly worker can take on a crucial role
```
Comment (IgorSusmelj) [inline docs/source/docker/integration/dagster_aws.rst:10]: Worker

File: docs/source/docker/integration/dagster_aws.rst [hunk 2]
```diff
@@ -2,33 +2,25 @@
...
-3. Run the Lightly Docker on the video to extract a diverse set of frames for further processing:
-   
-   a. Spin up an EC2 instance.
-   
-   b. Run the Lightly Docker
-   
-   c. Store the extracted frames in the S3 bucket
-   
-   d. Stop the EC2 instance
+3. Run the Lightly worker on the video to extract a diverse set of frames for further processing:
```
Comment (IgorSusmelj) [inline docs/source/docker/integration/dagster_aws.rst:17]: Worker

File: docs/source/docker/integration/dagster_aws.rst [hunk 3]
```diff
@@ -304,160 +295,63 @@ Set the `BUCKET_NAME` and `REGION_NAME` to your bucket name and region of the EC
...
 
-    REGION_NAME: str = 'YOUR_REGION_NAME' # e.g. eu-central-1
-    INSTANCE_ID: str = 'YOUR_INSTANCE_ID'
-    MOUNTED_DIR: str = '/home/ubuntu/lightly-aws-bucket'
 
 
-    class EC2Client:
-        """EC2 client to start, run, and stop instances.
+    class LightlyClient:
+        """Lightly client to run the Lightly worker.
```
Comment (IgorSusmelj) [inline docs/source/docker/integration/dagster_aws.rst:314]: Worker

File: docs/source/docker/integration/dagster_aws.rst [hunk 4]
```diff
@@ -304,160 +295,63 @@ Set the `BUCKET_NAME` and `REGION_NAME` to your bucket name and region of the EC
...
-            print(f'{wait_for}: OK')
-
+        def __init__(self, token: str, dataset_id: str):
+            self.token = token
+            self.dataset_id = dataset_id
 
-        def start_instance(self, instance_id: str):
-            """Starts the EC2 instance with the given id.
+        def run_lightly_worker():
+            """Runs the Lightly worker on the EC2 instance.
```
Comment (IgorSusmelj) [inline docs/source/docker/integration/dagster_aws.rst:323]: Worker

File: docs/source/docker/integration/dagster_aws.rst [hunk 5]
```diff
@@ -304,160 +295,63 @@ Set the `BUCKET_NAME` and `REGION_NAME` to your bucket name and region of the EC
...
 
     @solid
-    def run_lightly_onprem(object_name: str) -> None:
-        """Dagster solid to run Lightly On-premise on a remote EC2 instance.
-
-        Args:
-            object_name:
-                S3 object containing the input video(s) for Lightly.
+    def run_lightly_worker() -> None:
+        """Dagster solid to run Lightly worker on a remote EC2 instance.
```
Comment (IgorSusmelj) [inline docs/source/docker/integration/dagster_aws.rst:349]: Worker

**Review by IgorSusmelj — APPROVED** [review]
Just some renaming. Otherwise looks good to me :)

File: docs/source/docker/integration/dagster_aws.rst [hunk 6]
```diff
@@ -38,89 +30,88 @@ Dagster is an open-source data orchestrator for machine learning. It enables bui
...
+    from lightly.openapi_generated.swagger_client.models.dataset_type import DatasetType
+    from lightly.openapi_generated.swagger_client.models.datasource_purpose import DatasourcePurpose
+
+    # Create the Lightly client to connect to the API.
+    client = ApiWorkflowClient(token="YOUR_LIGHTLY_TOKEN")
+
+    # Create a new dataset on the Lightly Platform.
+    client.create_new_dataset_with_unique_name(
+        'my-datapool-name',
+        DatasetType.IMAGES  # can be DatasetType.VIDEOS when working with videos
```
Comment (MalteEbner) [inline docs/source/docker/integration/dagster_aws.rst:59]: "can be" -> "must be"

File: docs/source/docker/integration/dagster_aws.rst [hunk 7]
```diff
@@ -2,33 +2,25 @@
...
-3. Run the Lightly Docker on the video to extract a diverse set of frames for further processing:
-   
-   a. Spin up an EC2 instance.
-   
-   b. Run the Lightly Docker
-   
-   c. Store the extracted frames in the S3 bucket
-   
-   d. Stop the EC2 instance
+3. Run the Lightly Worker on the video to extract a diverse set of frames for further processing:
```
Comment (MalteEbner) [inline docs/source/docker/integration/dagster_aws.rst:17]: Why the colon at the end?

File: docs/source/docker/integration/dagster_aws.rst [hunk 8]
```diff
@@ -38,89 +30,88 @@ Dagster is an open-source data orchestrator for machine learning. It enables bui
...
+-----------------------------
+The next step is to set up the EC2 instance. For the purposes of this tutorial,
+it's recommended to pick an instance with a GPU (like the g4dn.xlarge) and the "Deep Learning AMI (Ubuntu 18.04) Version 48.0" AMI.
+See `this guide <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EC2_GetStarted.html>`_ to get started. Connect to the instance.
 
-.. code-block:: console
 
-    s3fs#lightly-aws-bucket /home/ubuntu/lightly-aws-bucket/ fuse _netdev,allow_other,umask=000,passwd_file=/home/ubuntu/.passwd-s3fs 0 0
+Next, the Lightly Worker should be installed on the instance. Please follow the instructions `here <https://docs.lightly.ai/docker/getting_started/setup.html>`__.
+Make sure you have the API token and the worker id from the setup steps. Start the worker in waiting mode with the following arguments:
```
Comment (MalteEbner) [inline docs/source/docker/integration/dagster_aws.rst:95]: Perhaps you could tell to put this in a bash script.

File: docs/source/docker/integration/dagster_aws.rst [hunk 9]
```diff
@@ -484,8 +378,8 @@ To put the solids together in a single pipeline, save the following code in `aws
         
         """
         file_name = download_random_video_from_pexels()
-        object_name = upload_video_to_s3(file_name)
-        run_lightly_onprem(object_name)
+        upload_video_to_s3(file_name)
+        run_lightly_onprem()
```
Comment (MalteEbner) [inline docs/source/docker/integration/dagster_aws.rst:382]: I was wondering why there is no argument passed to running lightly on prem. Perhaps you could tell that lightly analyses all videos in the bucket, not only the one just uploaded?

**Review by MalteEbner — COMMENTED** [review]
Overall, I like a lot that we streamline the workflow. I have some minor comments.

**Pull request merged**